main.js

```diff
@@ -20,6 +20,7 @@ const cli = meow(`
 	  --ignore          Additional paths to ignore  [Can be set multiple times]
 	  --space           Use space indent instead of tabs  [Default: 2]
 	  --no-semicolon    Prevent use of semicolons
+	  --prettier        Format code with Prettier
```

Comment:
Also add it to the `flags` option: https://github.com/sindresorhus/xo/blob/bc66114d3381e24b4272dfd88673643a9cff249e/main.js#L47

Reply:
And should we maybe clarify that the flag itself doesn't format the code but should be used with the `--fix` flag?

Reply:
I clarified the option description with `Conform to Prettier code style` as it does two things:
- report errors if the code is not formatted according to Prettier format
- automatically format with --fix (as with any other rules)

Let me know if that's clear enough.

I also added the flag